entity-management-web/src/main/java/eu/europeana/entitymanagement/web/JobLauncherController.java

```diff
@@ -0,0 +1,46 @@
+package eu.europeana.entitymanagement.web;
+
+import eu.europeana.entitymanagement.batch.DummyJobConfigurer;
+import eu.europeana.entitymanagement.batch.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.configuration.annotation.BatchConfigurer;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+
+/**
+ * Temporary controller to test Spring Batch integration.
+ * Functionality will be moved to {@link EMController}
+ */
+@RestController
+@RequestMapping(path = "/jobs")
+public class JobLauncherController {
+
+    @Autowired
+    private DummyJobConfigurer dummyJobConfigurer;
```

Comment:
if this is really used, I suggest to give the class an appropriate name

Reply:
As mentioned in the class comment, this is a temporary controller used to test the Spring Batch integration.

It will be removed once we start implementing proper Spring Batch jobs.